libs/core/src/lib/splitter/splitter-pane-container/splitter-pane-container.component.ts

```diff
@@ -0,0 +1,439 @@
+import { DOCUMENT } from '@angular/common';
+import {
+    AfterContentInit,
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ContentChildren,
+    ElementRef,
+    EventEmitter,
+    Inject,
+    Input,
+    OnDestroy,
+    Optional,
+    Output,
+    QueryList,
+    SkipSelf,
+    ViewEncapsulation
+} from '@angular/core';
+import { fromEvent, Subscription } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+import { RtlService } from '@fundamental-ngx/core/utils';
+
+import { SplitterPaneResizeEvent } from '../interfaces/splitter-pane-resize-event.interface';
+import { SplitterSplitPaneComponent } from '../splitter-split-pane/splitter-split-pane.component';
+import { PANE_AUTO_SIZE, PANE_NONE_SIZE, RESIZER_SIZE_PX, ROOT_PAGE } from '../constants';
+import { SplitterComponent } from '../splitter.component';
+
+
+export enum SplitterPaneContainerOrientation {
+    vertical = 'vertical',
+    horizontal = 'horizontal'
+}
+
+/** @dynamic */
```

Comment:
why do you add @dynamic ?

Reply:
Because of using `@Inject(DOCUMENT)`. Won't build without this.